app/websocket/app/main/model_manager.py

```diff
@@ -75,9 +91,21 @@ def _remove_model_from_cache(model_id: str):
 # ============== DB related functions =================
 
 
-def _save_model_in_db(serialized_model: bytes, model_id: str):
+def _save_model_in_db(
+    serialized_model: bytes,
+    model_id: str,
+    allow_model_copy: bool,
+    allow_run_inference: bool,
+):
     db.session.remove()
-    db.session.add(TorchModel(id=model_id, model=serialized_model))
+    db.session.add(
```

Comment:
add try except here too? And on exception just make it raise the exception... 
Or in the db function descriptions we can add can raise exceptions.

Reply:
Not needed, the exception will caught `save_model`. Only place where this function is used.